### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone or `ldd`. I don't think `@@` is understood outside of Mayhemfile contexts.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+image: ghcr.io/capuanob/trimesh2-fuzz:latest
```
Comment (xansec) [inline Mayhemfile:1]: Please omit image name.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Done!

**Pull request merged**